app/views/practices/overview.html.erb

```diff
@@ -39,6 +39,37 @@
               <%= f.text_area :tagline, class: "usa-textarea #{ @practice.errors[:tagline].any? ? 'usa-input--error' : '' } display-block practice-editor-tagline-textarea" %>
               <p class="usa-error-message <%= @practice.errors[:tagline].any? ? 'fas fa-exclamation-circle fon' : 'display-none' %>">&nbsp;<span class="font-family-sans"><%= show_errors(@practice, :tagline) %></span></p>
             </div>
+            <p class="margin-top-3"><span class="text-bold">Practice thumbnail:</span>&nbsp;</p>
+            <div class="grid-row margin-top-1">
+              <div class="grid-col-6">
+                <% if @practice.main_display_image.present? %>
+                  <%= image_tag @practice.main_display_image.url, alt: "practice thumbnail for #{@practice.name}", class: 'height-mobile practice-editor-impact-photo radius-md' %>
```

Comment:
ok, so, something funky about DM, and being in the VAEC, is our S3 bucket is not public readable at all.
that means we cannot get our images the "normal" way by just using a "normal" image url.
instead, we have to use the `presigned_url` and send it to S3 to let it know that we're who we say we are and can read files n stuff.
the way to do that is like this 

`@practice.main_display_image_s3_presigned_url` 

```suggestion
                  <%= image_tag @practice.main_display_image_s3_presigned_url, alt: "practice thumbnail for #{@practice.name}", class: 'height-mobile practice-editor-impact-photo radius-md' %>
```
like so.
keep in mind for future images and files that we retrieve from s3. it'll make your life easier and i forget all the time too so it'll make my life easier XD haha

Reply:
ooo ok! Thank you for letting me know! I shall make that change :D